design/images/noosphere-dark.svg

```diff
@@ -0,0 +1 @@
27.59-20,27.59-11.46,0-15.28-7.64-16.56-16.55-.84-6.79-1.69-14-6.79-14-7.64,0-13.15,8.07-17,17.83-3.82,10.18-6,19.95-6,27.16v87.43c0,18.68,1.7,23.77,15.28,25.89,11,2.12,15.28,3,15.28,6.79,0,4.25-3,5.1-10.18,5.1-4.67,0-10.61-1.7-37.78-1.7-21.64,0-27.58,1.7-32.25,1.7-5.52,0-8.92-1.28-8.92-5.1,0-4.24,3.82-5.09,10.62-6.79,10.18-1.27,12.3-3.39,12.3-19.95Z"/><path class="cls-9" d="M1817.93,298.94c-7.64,0-10.61,3-10.61,9.76,0,43.72,18.68,77.67,48,77.67,19.53,0,31-9.76,36.93-24.19,3.39-7.64,4.24-13.16,7.64-13.16,2.54,0,4.24,1.27,4.24,5.94,0,6.37-2.12,14-5.94,21.22-8.49,18.25-28,31.84-52.2,31.84-48,0-76.82-40.75-76.82-97.2,0-70,35.65-105.68,76.39-105.68,34,0,58.57,25.47,58.57,68.33,0,23.77-.42,25.47-12.31,25.47Zm-9.76-23.34c0,8.06,1.28,10.18,16.13,10.18a141.65,141.65,0,0,0,20.8-1.7c12.73-1.69,21.22-5.09,23.34-8.48,3.4-3.4,4.67-8.07,4.67-17.83,0-25-11.46-40.32-28-40.32-21.65,0-36.93,28-36.93,57.72Z"/><path class="cls-10" d="M365.65,175.75a41,41,0,0,0,41-41,41,41,0,0,0,41,41,41,41,0,0,0-41,41A41,41,0,0,0,365.65,175.75Z"/><path class="cls-11" d="M230.28,403.5a133.67,133.67,0,0,0,73.51-21.91h-147A133.7,133.7,0,0,0,230.28,403.5Z"/><path class="cls-12" d="M134.25,363.08H326.3a134.58,134.58,0,0,0,17.75-22.48H116.51A134.52,134.52,0,0,0,134.25,363.08Z"/><path class="cls-13" d="M327.69,176.64H132.86a135.17,135.17,0,0,0-17.29,22.48H345A134.64,134.64,0,0,0,327.69,176.64Z"/><path class="cls-14" d="M106.76,322.1h247a133.21,133.21,0,0,0,7.34-22.49H99.42A134.12,134.12,0,0,0,106.76,322.1Z"/><path class="cls-15" d="M364.2,258.62H96.36c-.27,3.47-.45,7-.45,10.51,0,4,.21,8,.56,12H364.09c.35-3.94.56-7.93.56-12C364.65,265.59,364.47,262.09,364.2,258.62Z"/><path class="cls-16" d="M354.4,217.63H106.15a133.49,133.49,0,0,0-7.06,22.48H361.47A133.52,133.52,0,0,0,354.4,217.63Z"/><path class="cls-17" d="M245.5,135.65a134.46,134.46,0,0,0-90.94,22.48H306A133.61,133.61,0,0,0,245.5,135.65Z"/><path class="cls-18" d="M354.41,404.08a92.86,92.86,0,0,0,60.68-22.49H293.72A92.86,92.86,0,0,0,354.41,404.08Z"/><path class="cls-18" d="M354.41,404.08a92.86,92.86,0,0,0,60.68-22.49H293.72A92.86,92.86,0,0,0,354.41,404.08Z"/><path class="cls-19" d="M261.18,310.85a94.91,94.91,0,0,0,.7,11.25H446.93a90.67,90.67,0,0,0,0-22.49h-185A94.74,94.74,0,0,0,261.18,310.85Z"/><path class="cls-19" d="M261.18,310.85a94.91,94.91,0,0,0,.7,11.25H446.93a90.67,90.67,0,0,0,0-22.49h-185A94.74,94.74,0,0,0,261.18,310.85Z"/><path class="cls-20" d="M293.72,240.11H415.09a93.15,93.15,0,0,0-121.37,0Z"/><path class="cls-20" d="M293.72,240.11H415.09a93.15,93.15,0,0,0-121.37,0Z"/><path class="cls-21" d="M266.06,281.1h176.7a93.13,93.13,0,0,0-11.13-22.48H277.18A93.1,93.1,0,0,0,266.06,281.1Z"/><path class="cls-21" d="M266.06,281.1h176.7a93.13,93.13,0,0,0-11.13-22.48H277.18A93.1,93.1,0,0,0,266.06,281.1Z"/><path class="cls-22" d="M277.18,363.08H431.63a92.77,92.77,0,0,0,11.13-22.48H266.06A92.74,92.74,0,0,0,277.18,363.08Z"/><path class="cls-22" d="M277.18,363.08H431.63a92.77,92.77,0,0,0,11.13-22.48H266.06A92.74,92.74,0,0,0,277.18,363.08Z"/></svg>
```

Comment:
Love these -- maybe we can dump these out on e.g. `@subconsciousnetwork/branding` as well?

Reply:
https://github.com/subconsciousnetwork/noosphere/issues/149